guides/common/modules/proc_adding-upstream-repositories-for-deb.adoc

```diff
@@ -0,0 +1,43 @@
+[id="adding-upstream-repositories-for-{DL}-{DL-major-context}_{context}"]
+= Adding upstream repositories for {DL} {DL-major}
+
+This example creates a product and repositories for {DL} {DL-major}.
+
+.Prerequisites
+* xref:Extracting_GPG_Public_Key_Fingerprints_from_Release_Files_{context}[Extract the GPG public keys] from the `Release` file.
+* xref:Importing_a_Custom_GPG_Key_{context}[Import the content credentials] into {Project}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Click *Create Product* to create a product named `{DL} {DL-major}`.
+For more information, see xref:Creating_a_Custom_Product_{context}[].
```

Comment:
Lines 12 and 31 both talk about creating a product. I understand why you wouldn't want to include the xref in both steps. So how about including it as `.Additional resources` at the end of the procedure?

Reply:
Makes sense, applied.